src/Tribe/Promoter/View.php

```diff
@@ -53,14 +70,18 @@ public function display_auth_check_view() {
 			$auth_error    = ! $authorized;
 		}
 
-		$this->template( 'auth', array(
-			'authorized'   => $authorized,
-			'auth_error'   => $auth_error,
-			'logged_in'    => is_user_logged_in(),
-			'admin'        => $is_admin,
-			'promoter_key' => $promoter_key,
-			'license_key'  => $license_key,
-		), true );
+		if ( $authorized ) {
+			wp_redirect( esc_url_raw( $this->authorized_redirect_url() ) );
+		} else {
+			$this->template( 'auth', array(
```

Comment:
As we are touching this code it might be a good moment to move from `array()` into `[]` syntax.